packages/integration-sdk/CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ and this project adheres to
 
 ## Unreleased
 
+## 5.1.0 - 2020-12-08
```

Comment:
Is this up to date? I merged 3 PRs in this project today that I think will be included in this release.

Reply:
Definitely helpful to have PRs come through with their associated CHANGELOG.md changes! There are some things in the `### Added` section below. Anything missing from your 3 PRs?

Reply:
@aiwilliams it ended up just being a rebase issue.